src/Item.cpp

```diff
@@ -172,6 +179,21 @@ void cItem::FromJson(const Json::Value & a_Value)
 		m_CustomName = a_Value.get("Name", "").asString();
 		m_Lore = a_Value.get("Lore", "").asString();
 
+		int red = a_Value.get("Color_Red", -1).asInt();
```

Comment:
Where does the JSON come from? We should probably check for a value outside the range 0-255.